src/app/inventory/store/d2-store-factory.ts

```diff
@@ -22,56 +21,7 @@ const genderTypeToEnglish = {
   [DestinyGender.Male]: 'male',
   [DestinyGender.Female]: 'female',
   [DestinyGender.Unknown]: '',
-};
-
-// Prototype for Store objects - add methods to this to add them to all
-// stores.
-export const StoreProto = {
-  // Remove an item from this store. Returns whether it actually removed anything.
-  removeItem(this: DimStore, item: DimItem) {
-    // Completely remove the source item
-    const match = (i: DimItem) => item.index === i.index;
-    const sourceIndex = this.items.findIndex(match);
-    if (sourceIndex >= 0) {
-      this.items = [...this.items.slice(0, sourceIndex), ...this.items.slice(sourceIndex + 1)];
-
-      let bucketItems = this.buckets[item.location.hash];
-      const bucketIndex = bucketItems.findIndex(match);
-      bucketItems = [...bucketItems.slice(0, bucketIndex), ...bucketItems.slice(bucketIndex + 1)];
-      this.buckets[item.location.hash] = bucketItems;
-
-      if (
-        this.current &&
-        item.location.accountWide &&
-        this.vault &&
-        this.vault.vaultCounts[item.location.hash]
-      ) {
-        this.vault.vaultCounts[item.location.hash].count--;
-      }
-
-      return true;
-    }
-    return false;
-  },
-
-  addItem(this: DimStore, item: DimItem) {
-    this.items = [...this.items, item];
-    this.buckets[item.location.hash] = [...this.buckets[item.location.hash], item];
-    item.owner = this.id;
-
-    if (this.current && item.location.accountWide && this.vault) {
-      this.vault.vaultCounts[item.location.hash].count++;
-    }
-  },
-
-  isDestiny1(this: DimStore) {
-    return false;
-  },
-
-  isDestiny2(this: DimStore) {
-    return true;
-  },
-};
+} as const;
```

Comment:
oh haven't seen that before, just looked it up. neat!

saw that "Outside of .tsx files, the angle bracket assertion syntax can also be used."
```
// Type '"hello"'
let x = <const>"hello";
```
Crazy, haven't seen that before.